server/lib/chat/chat-models.ts

```diff
@@ -199,6 +204,58 @@ export async function createChannel(
   }
 }
 
+/**
+ * Updates the channel with the given list of updates. Throws an error if there are no updates, so
+ * make sure to only call this method when you update one of the channel's fields.
+ */
+export async function updateChannel(
+  channelId: SbChannelId,
+  updates: Patch<EditChannelRequest>,
+  withClient?: DbClient,
+): Promise<FullChannelInfo> {
+  const { client, done } = await db(withClient)
+  try {
+    let query = sql`
```

Comment:
After you rearrange `updateEntries`, you can construct the whole query in one go instead of appending multiple times